tutorials/export/exporting_pcks.rst

```diff
@@ -1,7 +1,14 @@
 .. _doc_exporting_pcks:
 
-Exporting packs, patches, and mods
-==================================
+Exporting packs and patches
+===========================
+
+.. warning::
```

Comment:
```suggestion
.. danger::
```
Since the docs use `warning` for so many things, we should use `danger` for anything that is a legal, security, or data loss risk. See https://github.com/godotengine/godot-docs/pull/10252.